tools/workflow_schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/i-am-bee/bee-hive/workflow.schema.json",
+  "title": "Bee Hive",
+  "description": "A schema for defining Bee Hive workflows in YAML or JSON",
+  "type": "object",
+  "metadata": {
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string",
+        "description": "workflow name"
+      },
+      "labels": {
+        "type": "object",
+        "description": "workflow labels, key: value pairs"
+      }
+    },
+    "required": [
+      "name"
+    ]
+  },
+  "spec": {
+    "strategy": {
+      "type": "sequence"
+    },
+    "properties": {
+      "name": {
+        "type": "string",
+        "description": "workflow name"
+      },
+      "labels": {
+        "type": "object",
+        "description": "workflow labels, key: value pairs"
+      }
+    }
+  },
+  "agents": {
+    "type": "array",
+    "items": {
+      "type": "string"
+    }
+  },
+  "steps": {
+    "type": "object"
+  },
```

Comment:
should this also be an array?

Reply:
No, it should not.  It's a dictionary so just it's object.

Reply:
solved. Thx